### ISSUE TITLE
parse expensify csv output into expenses

### ISSUE THREAD

**Issue by sbillinge** [regro/regolith#199]
Here is an xlsx version of the file
[ACA_Toronto.xlsx](https://github.com/regro/regolith/files/2238850/ACA_Toronto.xlsx)

Notes: 
1. expensify saves it as a csv, I just couldn't attache a csv to GitHub for some funny reason
1. there is a reimbursable column and some expenses are repeated, as two expenses with amounts tagged as reimbursable or non-reimbursable.  It is how expensify handles segregated expenses.  We will hvae to have a way of "guessing" which are the same expense but with segregated and unsegregated parts
1. to begin with, we could not worry about the foreign amounts.  That could be in a later PR
1. we need some kind of reasonable workflow for making this work.  There will be missing info, such as payee etc.,

**Comment by CJ-Wright** [comment]
I think `ingest` would be a good place for this in the CLI. Would it be possible to ask expensify to give us the CSV via an API?

I think the missing info should be pulled in from the user config/rc/cli args. This way the payee could be controlled by user config (since the same computer most likely does the same person's account), the project, purpose, type can be from the CLI (since this could change).

**Comment by sbillinge** [comment]
there is an API.  Each individual will need their own user ID and secret
though.  If we want regolith to auto-build these, we may have to store this
info somewhere which is a bit awkward.  Let's think a bit about the right
workflow.

**Comment by CJ-Wright** [comment]
Ok, we could have each user store their config info in their `.config/regolith.yaml` file. That would allow each user to save their config in a place that others don't have access to.

**Comment by sbillinge** [comment]
can regolith scoop up all these when it builds something though?

**Comment by CJ-Wright** [comment]
It gets the user config as part of loading the rc (although it would only have one user config at a time). We could also store this info in the private database so we would have a central store of all the accounts.

But maybe we're over thinking this, we could produce a simple helper tool which gets a person's csv (or takes an existing csv) and turns it into valid yaml. Then it would be on the user to copy it over to the database and fill in any additional info. This is a bit less automagic than I'd like, but it does allow us to build core functionality before building an API system (and we can always add that later, it might be good to get adoption of the new system before backing it to the hilt).

**Comment by sbillinge** [comment]
Right. I was thinking sthg along these lines for the early workflow. Users
push their own downloaded csv to some holding repo and tegolith looks
there, ingests then deletes.

This way the users could put the expenses into a template google doc by
hand, or use Expensify, their choice. We would have to capture whose csv is
whose.

We need a good old UC for the workflow. Do you want to have a first crack
at it?

S
